internal/capability/types.go

```diff
@@ -3,26 +3,8 @@ package capability
 import (
 	"context"
 	"errors"
-	"time"
-
-	"github.com/opdev/opcap/internal/operator"
-	"github.com/operator-framework/api/pkg/operators/v1alpha1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-type options struct {
```

Comment:
Originally, this was used with the reports as a convenience. It would be better to leave it unexported here, and  expose the template data structs instead.

Reply:
agreed